packages/valory/skills/register_reset_recovery_abci/rounds.py

```diff
@@ -59,9 +58,6 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
             )
             all_round_counts.append(self.most_voted_payload)
             synchronized_data = self.synchronized_data.update(
-                synchronized_data_class=self.synchronized_data_class,
-                participants=self.collection,
-                all_participants=self.collection,
```

Comment:
Why removed?

Reply:
It wasn't necessary. It was copied over from a different test.